README.md

```diff
@@ -1,20 +1,12 @@
 # Exomiser Runner for PhEval
 
-This is the Exomiser plugin for PhEval. Highly experimental. Do not use.
+This is the Exomiser plugin for PhEval.
```

Comment:
Would be good to have a link or brief explanation on how the plugin relates to pheval and how that needs to be set-up, assuming it needs some kind of setup before running this.